menu/menu_displaylist.c

```diff
@@ -985,6 +989,96 @@ static unsigned menu_displaylist_parse_core_manager_list(
    return count;
 }
 
+#ifdef HAVE_MIST
+static unsigned menu_displaylist_core_manager_steam_list(
+      menu_displaylist_info_t *info,
+      settings_t *settings)
+{
+   MistResult result;
+   steam_core_dlc_list_t *dlc_list;
+   steam_core_dlc_t *dlc_info;
+   unsigned count = 0;
+
+   result = steam_get_core_dlcs(&dlc_list, false);
+   if (MIST_IS_ERROR(result)) goto error;
+
+   for (size_t i = 0; dlc_list->count > i; i++)
+   {
+      dlc_info = steam_core_dlc_list_get(dlc_list, i);
+
+      if (menu_entries_append_enum(info->list,
+               dlc_info->name,
+               "",
+               MENU_ENUM_LABEL_CORE_MANAGER_STEAM_ENTRY,
+               MENU_SETTING_ACTION_CORE_MANAGER_STEAM_OPTIONS,
+               0, 0))
+      {
+         count++;
+      }
+   }
+
+   return count;
+   
+error:
+   /* TODO: Send error notification */
+   RARCH_ERR("[Steam] Error enumerating core dlcs for core manager (%d-%d)\n", MIST_UNPACK_RESULT(result));
+   return count;
+}
+
+static unsigned menu_displaylist_core_information_steam(
```

Comment:
For consistency, please name this `menu_displaylist_parse_core_information_steam()`